sys/include/net/gnrc/mac/internal.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *               2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_gnrc_mac
+ * @{
+ *
+ * @file
+ * @brief       Definitions of internal functions of GNRC_MAC module
+ *
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ * @author      Shuguo Zhuo  <shuguo.zhuo@inria.fr>
+ */
+
+#ifndef GNRC_MAC_INTERNAL_H_
+#define GNRC_MAC_INTERNAL_H_
+
+#include <stdint.h>
+#include <net/ieee802154.h>
+#include <net/gnrc/mac/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if GNRC_MAC_TX_QUEUE_SIZE != 0
+/* @brief Queues the packet into the related transmission packet queue in netdev2_t::tx.
```

Comment:
These are no doc-comments:

```C
/* @brief ...
```

vs.

```C
/**
 * @brief ...
```

Also consider adding `|| defined(DOXYGEN)` to the `#if` above so DOXYGEN picks up on this